src/ralph/networks/templates/networks/network_change_list.html

```diff
@@ -0,0 +1,13 @@
+{% extends 'admin/change_list.html' %}
+{% block breadcrumbs %}
+<ul class="breadcrumbs">
+  <li><a href="{% url 'admin:networks_network_changelist' %}">Network</a></li>
```

Comment:
this is not the original behaviour of https://github.com/allegro/ralph/blob/ng/src/ralph/admin/templates/admin/breadcrumbs_foundation.html - couldn't you wrap the for loop here in some block and overwrite it with calling super and adding parent networks after default content?